src/main/java/xyz/nikitacartes/easyauth/storage/database/MongoDB.java

```diff
@@ -23,9 +23,9 @@
 
     public static void initialize() {
         mongoClient = MongoClients.create(
-                config.main.MongoDBConnectionString
+                "mongodb://" + config.main.databaseUser + ":" + config.main.databasePassword + "@" + config.main.databaseHost + "/" + config.main.databaseConnectionOptions
```

Comment:
Some MongoDB connections `mongodb+srv://` so it wouldn't work.

Reply:
Changed in https://github.com/NikitaCartes/EasyAuth/pull/34/commits/97defe20e9fbe2139c1ae3f71fd904235b94a5d2

Reply:
> Some MongoDB connections looks like `mongodb+srv://` so it wouldn't work. I would leave `MongoDBConnectionString` as is and add options for MySQL but it's up to you. (I will rewrite all config stuff anyway, so you can do it in a quick and dirty way for now)

Changed in https://github.com/NikitaCartes/EasyAuth/pull/34/commits/97defe20e9fbe2139c1ae3f71fd904235b94a5d2